docs/azure-tre-overview/case-studies.md

```diff
@@ -4,17 +4,17 @@
 Imperial College London partnered with CloudKubed to address the increasing demands for scalable and centralised research solutions. The challenge was to enhance project visibility, streamline end-to-end research, and improve the management of research projects. CloudKubed implemented a cloud-based Azure Trusted Research Environment (TRE) aligned with Imperial’s strategy and needs.
 This TRE integrated approval processes and provided a secure, compliant environment for research. The outcome allowed Imperial to efficiently provision workspaces, facilitate access to research tools, and streamline the entire research process. The partnership significantly improved the efficiency of research operations and compliance with data security measures.
 
-Click [here](https://cloudkubed.com/case-studies/imperial-college-london-trusted-research-environment/ "Imperial College London TRE") to see the full case study.
+Click [here](https://cloudkubed.com/case-studies/imperial-college-london-trusted-research-environment/ "CloudKubed and Imperial College London"){target="_blank"} to see the full case study.
 
 ## Building the Trusted Research Environment with Azure Databricks
 The importance of secure and scalable data environments in healthcare research has grown, particularly highlighted by the COVID-19 pandemic. The Trusted Research Environments (TREs) enable secure collaboration on sensitive data while maintaining patient privacy. Azure Databricks enhances TREs by offering scalable, secure, and flexible data processing capabilities.
 For example, Providence Health used Azure Databricks to reduce hospital overcrowding and improve patient care by leveraging real-time data. The integration of Azure Databricks into the Azure TRE blueprint allows researchers to perform complex data analysis securely, promoting innovative research and improved healthcare outcomes.
```

Comment:
Can you remove the reference to Providence Health as not sure they actually use Azure TRE, just use Databricks .

Reply:
is this better?

Reply:
thanks.